src/current/_includes/v23.1/prod-deployment/prod-guidance-cache-max-sql-memory.md

```diff
@@ -1 +1,12 @@
-For production deployments, set `--cache` to `25%` or higher. Avoid setting `--cache` and `--max-sql-memory` to a combined value of more than 75% of a machine's total RAM. Doing so increases the risk of memory-related failures.
\ No newline at end of file
+{% capture formula %}{% include_cached copy-clipboard.html %}<div class="highlight"><pre><code class="language-none" data-lang="none">2 * --max-sql-memory + --cache &lt;= 80% of system RAM
+</code></pre></div>
+{% endcapture %}
+The default value for `--cache` is 128 MiB. For production deployments, set `--cache` to `25%` or higher. To determine appropriate settings for `--cache` and `--max-sql-memory`, use the following formula: {{ formula }}
+
+To help guard against OOM events, CockroachDB sets a soft memory limit using mechanisms in Go. Depending on your hardware and workload, you may not need to manually tune `--max-sql-memory`.
```

Comment:
'OOM events' could be a link to our docs on that: https://www.cockroachlabs.com/docs/v23.2/cluster-setup-troubleshooting#out-of-memory-oom-crash

(editing the link per-version ofc!)